core/src/main/scala/org/apache/spark/h2o/backends/internal/H2ORpcEndpoint.scala

```diff
@@ -62,3 +70,9 @@ case class StopEndpointMsg()
 case class FlatFileMsg(nodes: Array[NodeDesc], portOffset: Int)
 
 case class StartH2OWorkersMsg(conf: H2OConf)
+
+case class CheckClusterSizeMsg()
```

Comment:
nit Couldn't we use case objects instead of case classes here?

Reply:
Yup, will do